docs/source/status.rst

```diff
@@ -0,0 +1,199 @@
+| gRPC transport              | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+| gRPC + TLS transport        | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+| RPC error codes             | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+| Authentication handlers     | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+| Custom client middleware    | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+| Custom server middleware    | ✓     | ✓     |       |            |       |       |
++-----------------------------+-------+-------+-------+------------+-------+-------+
+
+.. seealso::
+   The :ref:`flight-rpc` specification.
+
+
+C Data Interface
+================
+
++-----------------------------+-------+--------+-------+
+| Feature                     | C++   | Python | R     |
+|                             |       |        |       |
++=============================+=======+========+=======+
+| Schema export               | ✓     | ✓      | ✓     |
++-----------------------------+-------+--------+-------+
+| Array export                | ✓     | ✓      | ✓     |
++-----------------------------+-------+--------+-------+
+| Schema import               | ✓     | ✓      | ✓     |
++-----------------------------+-------+--------+-------+
+| Array import                | ✓     | ✓      | ✓     |
++-----------------------------+-------+--------+-------+
+
+.. seealso::
+   The :ref:`C Data Interface <c-data-interface>` specification.
+
+
+Third-Party Data Formats
+========================
+
++-----------------------------+---------+---------+-------+------------+-------+-------+
+| Format                      | C++     | Java    | Go    | JavaScript | C#    | Rust  |
+|                             |         |         |       |            |       |       |
++=============================+=========+=========+=======+============+=======+=======+
+| Avro                        |         | R       |       |            |       |       |
++-----------------------------+---------+---------+-------+------------+-------+-------+
+| CSV                         | R       |         | R/W   |            |       | R/W   |
++-----------------------------+---------+---------+-------+------------+-------+-------+
+| ORC                         | R       |         |       |            |       |       |
++-----------------------------+---------+---------+-------+------------+-------+-------+
+| Parquet                     | R/W     |         |       |            |       |       |
```

Comment:
Should we note the nested type parquet limitation?

Reply:
Every reader/writer implementation probably has limitations. I don't think it's worth detailing them here.